app/Czech.php

```diff
@@ -87,6 +38,15 @@ class Czech implements ValidatorInterface
     ];
 
     /**
+     * @param string $cnfFile
+     * @throws MissingBankCodesFileFile
+     */
+    public function __construct($cnfFile = __DIR__ . '/../cnf/czech-bank-codes.csv')
+    {
+        $this->validBankCodes = $this->getBankCodes($cnfFile);
```

Comment:
It would. But it is really necessary now? My time is limited for more changes :/ So we could wait or you can prepare PR ;)